### PR TITLE
Review UnitTestBot Go README and Developer guide for linguistic and formatting bugs

### PR DESCRIPTION
**Pull request by olganaumenko**
## Description

Linguistic bugs fixed, document structure optimized.

## How to test

The PR introduces no code changes.
Manual scenario: the text itself has been previewed in the IntelliJ IDEA viewer.

## Self-check list

Check off the item if the statement is true. Hint: [x] is a marked item.

Please do not delete the list or its items.

- [x] I've set the proper **labels** for my PR (at least, for category and component).
- [x] PR **title** and **description** are clear and intelligible.
- [ ] I've added enough **comments** to my code, particularly in hard-to-understand areas.
- [ ] The functionality I've repaired, changed or added is covered with **automated tests**.
- [x] **Manual tests** have been provided optionally.
- [ ] The **documentation** for the functionality I've been working on is up-to-date.

### CODE REVIEW

File: utbot-go/README.md [hunk 1]
```diff
@@ -1,180 +1,141 @@
...
+* Go SDK 1.18 or later
+* Compatible [Go plugin](https://plugins.jetbrains.com/plugin/9568-go) for IntelliJ IDEA
+* Properly configured `go.mod` file for the code under test
+* `github.com/stretchr/testify/assert` and `golang.org/x/tools@v0.4.0` Go modules installed (IntelliJ IDEA automatically offers to install them as soon as the tests are generated)
 
-Most likely, if you are already developing Go project in IntelliJ IDEA, then you have already met all the requirements.
+#### Installation
 
-<ins>_To install the UTBot Go plugin in IntelliJ IDEA:_</ins>
+To install the UnitTestBot Go plugin in IntelliJ IDEA, refer to [UnitTestBot user guide](https://github.com/UnitTestBot/UTBotJava/actions/runs/3012565900).
```
Comment (Markoutte) [inline utbot-go/README.md:75]: Does the link lead to wrong place?

Comment (olganaumenko) [reply utbot-go/README.md]: Thank you! Fixed.

File: utbot-go/README.md [hunk 2]
```diff
@@ -1,180 +1,141 @@
...
-  version (it is for this that the `Ultimate` edition of the IDE is needed);
-* properly configured Go module for source code file (i.e. for file to generate tests for): corresponding `go.mod` file
-  must exist;
-* installed Go modules `github.com/stretchr/testify/assert` and `golang.org/x/tools@v0.4.0`: fortunately, IDEA will automatically highlight
-  it and offer to install the first time the tests are generated.
+* IntelliJ IDEA (Ultimate Edition) — versions from 2022.2 to 2022.2.4
+* Go SDK 1.18 or later
+* Compatible [Go plugin](https://plugins.jetbrains.com/plugin/9568-go) for IntelliJ IDEA
+* Properly configured `go.mod` file for the code under test
+* `github.com/stretchr/testify/assert` and `golang.org/x/tools@v0.4.0` Go modules installed (IntelliJ IDEA automatically offers to install them as soon as the tests are generated)
```
Comment (egiptipavel) [inline utbot-go/README.md:71]: It's my mistake. `golang.org/x/tools@v0.4.0` will be install automatically during source code analysis.

Comment (olganaumenko) [reply utbot-go/README.md]: Thank you! Fixed.

**Review by Markoutte — APPROVED** [review]

**Pull request merged**